javascript/code examples/nodejs.md

```diff
@@ -0,0 +1,18 @@
+## Browser
+Pushing file to lighthouse node using nodejs.
+
+```javascript
+// Deploy file
+const lighthouse = require('lighthouse-web3');
+
+const deploy = async() =>{
+  const path = "/mnt/c/Users/cosmos/Desktop/projects/testing/status.js";	//Give absolute path
+  const apiKey = process.env.API_KEY;
+
+  const response = await lighthouse.deploy(path, apiKey);
+  // Display response
+  console.log(response);
```

Comment:
put in here an example response output. This will help devs to identify if they get the correct output. (ideally this should return CID + other info)